web/src/services/loans.service.js

```diff
@@ -238,17 +238,42 @@ const getStudentLoans = () => {
 };
 
 const cancelLoan = (id) => {
-  return instance.put(`/cancel/${id}/`).then((response) => {
-    console.log("Canceled: ", response.data.message);
-    return true;
-  });
+  return instance
+    .put(`/cancel/${id}/`)
+    .then((response) => {
+      console.log("Canceled: ", response.data.message);
+      return true;
+    })
+    .catch((error) => {
+      console.log(error.response);
+      return false;
+    });
 };
 
 const withdrawLoanRequest = (id) => {
-  return instance.put(`/user_withdraw/${id}/`).then((response) => {
-    console.log("Withdrawn: ", response.data.message);
-    return true;
-  });
+  return instance
+    .put(`/user_withdraw/${id}/`)
+    .then((response) => {
+      console.log("Withdrawn: ", response.data.message);
+      return true;
+    })
+    .catch((error) => {
+      console.log(error.response);
+      return false;
+    });
+};
```

Comment:
cancel and withdrawn were throwing errors for ManageLoan to catch, and work with them